src/proxy.c

```diff
@@ -1082,15 +1082,15 @@ cc_proxy_hyper_pod_create (struct cc_oci_config *config)
 	JsonObject                   *ipaddr_obj = NULL;
 	JsonArray                    *ipaddr_arr = NULL;
 
-	if (! (config && config->proxy)) {
+	if (! (config && config->proxy && config->net.hostname)) {
```

Comment:
@sameo Yes, the network setup runs before pod creation. 
https://github.com/01org/cc-oci-runtime/blob/576913626eeefd9d85cd028fe459a9d666f076f5/src/networking.c#L551